pkg/controller/logstash/pod.go

```diff
@@ -127,3 +129,8 @@ func readinessProbe(useTLS bool) corev1.Probe {
 		},
 	}
 }
+
+// GetLogstashContainer returns the Logstash container from the given podSpec.
+func GetLogstashContainer(podSpec corev1.PodSpec) *corev1.Container {
+	return pod.ContainerByName(podSpec, "logstash")
```

Comment:
there is a constant for the container name https://github.com/elastic/cloud-on-k8s/blob/feature/logstash/pkg/apis/logstash/v1alpha1/logstash_types.go#L17